src/test/java/wooteco/subway/dao/LineDaoTest.java

```diff
@@ -64,7 +64,7 @@ public void findAll() {
     @DisplayName("노선을 하나 추가한 뒤, 전체 노선을 조회한다")
     public void findAll_afterSaveOneLine() {
         // given
-        linDao.save(new Line(LINE_NAME, LINE_COLOR, section));
+        linDao.save(new Line(LINE_NAME, LINE_COLOR,900, section));
```

Comment:
```suggestion
        linDao.save(new Line(LINE_NAME, LINE_COLOR, 900, section));
```

매의 눈 @_@ | 👁️‍🗨️_👁️‍🗨️